src/r5py/r5/transport_network.py

```diff
@@ -62,6 +64,47 @@ def __init__(self, osm_pbf, gtfs=[], build_config={}):
         )
         self._transport_network.transitLayer.buildDistanceTables(None)
 
+    @classmethod
+    def from_directory(cls, path):
+        """
+        Load a transport network, find input data in `path`.
```

Comment:
Or "Find input data in `path` to load a transport network for r5py." ?

Reply:
I would maybe leave the r5py out there, or make it ’... load an `r5py.TransportNetwork`’ - does that sound okay?

Reply:
I think that sounds good and explicit 👍🏻